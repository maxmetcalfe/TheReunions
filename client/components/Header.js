import React, { Component } from "react";

class Header extends Component {

  render() {
      return (
        <div>
          <h1>Welcome to The Reunions.</h1>
          <h2>
            A group of four friends travel together around the world.
            This is where they have been.
          </h2>
        </div>
      );
  }
}

export default Header;
