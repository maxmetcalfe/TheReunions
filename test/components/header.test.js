import React from "react"
import renderer from "react-test-renderer"
import Header from "../../src/components/Header"

test("Header", () => {
  let component = renderer.create(<Header></Header>)

  expect(component).toMatchSnapshot()
})
