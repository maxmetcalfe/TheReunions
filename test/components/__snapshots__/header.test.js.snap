// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<div>
  <h1>
    Welcome to The Reunions.
  </h1>
  <h2>
    A group of four friends travel together around the world. This is where they have been.
  </h2>
</div>
`;
